release.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+// Release contains details from release.json
+type Release struct {
+	Release       string               `json:"release"`
+	Stream        string               `json:"stream"`
+	Metadata      Metadata             `json:"metadata"`
+	Architectures ReleaseArchitectures `json:"architectures"`
```

Comment:
Might be worthwhile to use a [map of strings to a generic architecture type](https://github.com/coreos/mantle/blob/master/cmd/plume/types.go#L104-L106) over a wrapper struct.